package.json

```diff
@@ -1,155 +1,132 @@
+    "type": "git",
+    "url": "git@github.com:yahoo/react-intl.git"
+  },
+  "main": "./dist/index.js",
+  "module": "./lib/index.js",
+  "browserify-shim": {
+    "react": "global:React"
+  },
+  "dependencies": {
+    "hoist-non-react-statics": "^3.3.0",
+    "intl-format-cache": "^3.0.0",
+    "intl-locales-supported": "^1.0.10",
+    "intl-messageformat": "^4.0.0",
+    "intl-relativeformat": "^6.1.0",
+    "invariant": "^2.1.1",
+    "react-display-name": "^0.2.4",
+    "react-is": "^16.3.1"
+  },
+  "peerDependencies": {
+    "prop-types": "^15.5.4",
+    "react": "^16.3.1"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/node": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-transform-async-to-generator": "^7.1.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.1.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "@formatjs/intl-relativetimeformat": "^1.0.0",
+    "babel-eslint": "^7.1.1",
+    "babel-jest": "^24.0.0",
+    "babel-plugin-react-intl": "^3.0.0",
+    "babel-plugin-transform-member-expression-literals": "^6.9.4",
+    "babel-plugin-transform-property-literals": "^6.9.4",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.18",
+    "babelify": "^7.2.0",
+    "benchmark": "^2.1.0",
+    "browserify": "^14.0.0",
+    "browserify-shim": "^3.8.11",
+    "cross-env": "^4.0.0",
+    "enzyme": "^3.6.0",
+    "enzyme-adapter-react-16": "^1.5.0",
+    "eslint": "^3.10.2",
+    "eslint-plugin-react": "^7.0.1",
+    "expect": "^1.9.0",
+    "express": "^4.13.3",
+    "formatjs-extract-cldr-data": "^6.0.0",
+    "full-icu": "^1.3.0",
+    "glob": "^7.0.0",
+    "intl": "^1.2.1",
+    "intl-messageformat-parser": "^1.2.0",
+    "intl-pluralrules": "^1.0.1",
+    "jest": "^24.0.0",
+    "mkdirp": "^0.5.1",
+    "prettier": "^1.6.1",
+    "progress": "^2.0.3",
+    "promise-queue": "^2.2.5",
+    "prop-types": "^15.6.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "rimraf": "^2.4.2",
+    "rollup": "^1.13.0",
+    "rollup-plugin-babel": "^4.3.2",
+    "rollup-plugin-commonjs": "^10.0.0",
+    "rollup-plugin-node-resolve": "^5.0.1",
+    "rollup-plugin-re": "^1.0.7",
+    "rollup-plugin-replace": "^2.0.0",
+    "rollup-plugin-typescript": "^1.0.1",
+    "rollup-plugin-uglify": "^6.0.0",
+    "rollup-plugin-virtual": "^1.0.1",
+    "serialize-javascript": "^1.5.0",
+    "superagent": "^3.0.0",
+    "ts-jest": "^24.0.2",
+    "tslib": "^1.9.3",
+    "typescript": "^3.5.1",
+    "watchify": "^3.7.0"
+  },
+  "scripts": {
+    "format": "prettier --check *.js scripts/*.js src/{*.js,**/*.js} *.md **/*.md",
+    "format:fix": "prettier --write *.js scripts/*.js src/{*.js,**/*.js} *.md **/*.md",
+    "clean": "rimraf src/en.js coverage/ dist/ lib/ test/renderer.js",
+    "build:lib": "tsc && tsc -p tsconfig.cjs.json",

```

Comment:
how long does this take running it twice? in development we should add a watcher `dev:watch` or something to make it easier to develop

Reply:
TS compiler already supports incremental build so it's pretty quick as is